src/core_modules/capture-core/components/DataEntries/EnrollmentRegistrationEntry/EnrollmentRegistrationEntry.component.js

```diff
@@ -1,22 +1,32 @@
 // @flow
 import React, { type ComponentType } from 'react';
 import { Button } from '@dhis2/ui';
+import i18n from '@dhis2/d2-i18n';
 import { withStyles } from '@material-ui/core';
 import { compose } from 'redux';
 import { useScopeInfo } from '../../../hooks/useScopeInfo';
 import { scopeTypes } from '../../../metaData';
 import { EnrollmentDataEntry } from '../Enrollment';
 import { useCurrentOrgUnitInfo } from '../../../hooks/useCurrentOrgUnitInfo';
 import { useRegistrationFormInfoForSelectedScope } from '../common/useRegistrationFormInfoForSelectedScope';
-import type { OwnProps } from './EnrollmentRegistrationEntry.types';
+import type { Props } from './EnrollmentRegistrationEntry.types';
 import { withSaveHandler } from '../../DataEntry';
+import { withLoadingIndicator } from '../../../HOC';
+import { InfoIconText } from '../../InfoIconText';
 
 const styles = ({ typography }) => ({
     marginTop: {
         marginTop: typography.pxToRem(2),
     },
 });
 
+const translatedTextWithStylesForProgram = (trackedEntityName: string, programName: string, orgUnitName: string) =>
+    (<>
+        {i18n.t('Saving a {{trackedEntityName}} in', { trackedEntityName })} <b>{programName}</b>
+        {orgUnitName && <>{' '}{i18n.t('in')} <b>{orgUnitName}</b></>}.
+    </>);
```

Comment:
Shall we make a ticket for this @JoakimSM?

Reply:
Yeah, perfect if you make a ticket for this.

Reply:
I did. Can you fill in the details please? 
https://jira.dhis2.org/browse/DHIS2-10267 I dont really know what was the problem on this one. I think you know best